src/Approvals/SFA.DAS.Approvals.Api/Controllers/CohortController.cs

```diff
@@ -150,6 +151,30 @@ public async Task<IActionResult> GetAddDraftApprenticeshipCourse([FromQuery] lon
             }
         }
 
+        [HttpGet]
+        [Route("employer/{accountId}/unapproved/add/confirm-employer")]
```

Comment:
I don't think we'd ever need this endpoint from the employer UI, since the "confirm employer" page doesn't exist.

Reply:
Done